pytorch_lightning/logging/tensorboard.py

```diff
@@ -89,6 +95,17 @@ def save(self):
             # you are using PT version (<v1.2) which does not have implemented flush
             self.experiment._get_file_writer().flush()
 
+        # create a preudo standard path ala test-tube
+        dir_path = os.path.join(self.save_dir, self.name, 'version_%s' % self.version)
+        if not os.path.isdir(dir_path):
+            dir_path = self.save_dir
+        # prepare the file path
+        meta_tags_path = os.path.join(dir_path, self.NAME_CSV_TAGS)
+        # save the metatags file
+        df = pd.DataFrame({'key': list(self.tags.keys()),
+                           'value': list(self.tags.values())})
+        df.to_csv(meta_tags_path, index=False)
```

Comment:
Shouldn't all this tag saving stuff go into the `log_hyperparams` function, not `log_metrics`?

Reply:
good question, what do you propose? @williamFalcon

Reply:
I think you can just move all this new code up into where the warning is in `log_hyperparams`.

Reply:
agreed with @neggert

Reply:
so you want to do a save always when a new parameter is added?
it makes more sense for me to do it while saving...

Reply:
log_metrics gets called all the time. we shouldn’t save the csv over and over when that happens.

Reply:
so saving it in `save()` is fine? or am I missing something...

Reply:
i haven’t looked at this in a while. i would just put it where test tube does it

Reply:
test-tube does it while saving too, that's how I "copy-paste" it...
https://github.com/williamFalcon/test-tube/blob/master/test_tube/log.py#L366

Reply:
`log_hyperparams` is intended to be called once at the beginning of training. I think that's what we want.